string name
yolo_object_detection/BoundingBox box
float64 confidence